lib/manageiq/release/repo.rb

```diff
@@ -65,6 +65,10 @@ def remote?(remote)
         end
       end
 
+      def remote_branch?(remote, branch)
+        git.capturing.ls_remote(remote, branch).length > 0
```

Comment:
Alternately, `.present?` as opposed to `.length > 0`?

Reply:
ooh, we pull in activesupport?

I'll try it... that would be better